plugins/tiddlywiki/tiddlyweb/syncer-actions-refresh.tid

```diff
@@ -1,9 +1,17 @@
 title: $:/plugins/tiddlywiki/tiddlyweb/syncer-actions/refresh
 tags: $:/tags/SyncerDropdown
 
-<$reveal state="$:/status/IsLoggedIn" type="match" text="yes">
-<$button tooltip="Get latest changes from the server" aria-label="Refresh from server" class="tc-btn-invisible">
```

Comment:
@linonetwo In cases like this one it is better to use a transclusion. the `<<lingo` macro is only a helper, to have shorter references. But instead of using `wikify`-widget, which is slow, you can use a transclusion here
eg: 

```
<$button tooltip={{$:/plugins/tiddlywiki/tiddlyweb/language/Refresh/Label}} aria-label={{$:/plugins/tiddlywiki/tiddlyweb/language/Refresh/Label}}  class="tc-btn-invisible">
```

IMO this should work without the need for wikify

Reply:
But the tiddler title is `$:/plugins/tiddlywiki/tiddlyweb/language/en-GB/Refresh/Label`(with `/en-GB`) ... otherwise we need to create `$:/plugins/tiddlywiki/tiddlyweb/language/Refresh/Label`, and use lingo marco in it, but I suspect it will still transclude the `<<lingo` call as is, instead of wikify it.

Reply:
The solution maybe in another PR, write a new pragma, or rule, to disable this "Assigning Procedure Calls to Attribute Values" in https://tiddlywiki.com/#Procedure%20Parameter%20Handling